Program.cs

```diff
@@ -1,10 +1,15 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿Console.WriteLine (armstrong ());
+
+string armstrong () {
+   Console.Write ("Enter a number to check whether it is an armstrong number or not: ");
+   if (int.TryParse (Console.ReadLine (), out int input)) {
+      int temp = input;
+      double sum = 0;
+      while (temp > 0) {
+         double digit = temp % 10;
+         sum += Math.Pow (digit, input.ToString ().Length);

```

Comment:
Length of the number will not change after input. So, better to calculate length once. Here, length will be calculated every time the loop is executed.